examples/trace/build.gradle

```diff
@@ -45,4 +45,5 @@ dependencies {
 	compile(libraries.opentelemetry_sdk)
 	compile(libraries.google_cloud_trace)
 	compile project(':exporter-trace')
+	compile project(':detector-resources')
```

Comment:
Is this a dependency on the exporter or separately consumable thing?

Reply:
Even for this PR, it may help to change this to a `runtimeOnly` dependency and add a comment that it's loaded via `ServiceLoader`.